doc/src/fix_graphics_arrows.rst

```diff
@@ -0,0 +1,189 @@
+.. index:: fix graphics/arrows
+
+fix graphics/arrows command
+===========================
+
+Syntax
+""""""
+
+.. code-block:: LAMMPS
+
+   fix ID group-ID graphics/arrows Nevery mode keyword args ...
+
+* ID, group-ID are documented in :doc:`fix <fix>` command
+* graphics/arrows = style name of this fix command
+* Nevery = update graphics information every this many time steps
+* mode = one of the following modes *dipole* or *force* or *velocity* or *variable* or *chunk*
+
+  .. parsed-literal::
+
+     *dipole* args = scale radius
+       scale = scale factor for the dipole moment to determine the arrow length
+       radius = radius for arrows (length units)
+     *force* args = scale radius
+       scale = scale factor for the force vector to determine the arrow length
+       radius = radius for arrows (length units)
+     *velocity* args = scale radius
+       scale = scale factor for the velocity vector to determine the arrow length
+       radius = radius for arrows (length units)
+     *variable* args = xval yval zval radius
+       xval = x value for arrow vector (may be a variable)
+       yval = y value for arrow vector (may be a variable)
+       zval = z value for arrow vector (may be a variable)
+       radius = radius for arrows (length units)
+     *chunk* args = chunk-ID pos-ID vec-ID scale radius
+       chunk-ID = ID of :doc:`compute chunk/atom <compute_chunk_atom>` command
+       pos-ID = ID of a per-chunk compute that computes the positions for the arrows
+       vec-ID = ID of a per-chunk compute that computes the arrow vectors
+       scale = scale factor for the per-chunk vector to determine the arrow length
+       radius = radius for arrows (length units)
+
+* zero or more keyword/value pairs may be appended
+* keyword = *autoscale*
+
+  .. parsed-literal::
+
+       *autoscale* value = automatically scale arrows so they have an average length of "value"
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   fix vec all graphics/arrows 10 velocity 20.0 0.066 autoscale 0.5
+   fix vec all graphics/arrows 100 variable v_xnorm v_znorm 0.0 0.066
+   fix vec all graphics/arrows 100 chunk molchunk com dip 1.0 0.05
+
+Description
+"""""""""""
+
+.. versionadded:: TBD
+
+This fix allows to add arrows to images rendered with :doc:`dump image
+<dump_image>` using the *fix* keyword to represent vector properties
+with arrows for either all atoms in the fix group or for :doc:`chunks
+<Howto_chunk>`.
+
+The *group-ID* sets the group ID of the atoms selected to have the selected
+property represented.  This may be a dynamic group.
+
+The *Nevery* keyword determines how often the arrows graphics data is
+updated.  This should be the same value as the corresponding *N*
```

Comment:
can the nevery keyword just be extracted from 'dump image', and removed from this fix?

Reply:
Hmmm... tricky, since the fix needs to be defined before the dump. Not sure if you can change it after the fact. Also, there is the idea to have the fix output VMD script commands to a file (not yet implemented) so the same kind of graphics (but at higher quality) can be created in VMD. I've already implemented a command for exporting region visualizations. I'm currently not making VMD a priority since the overall future of VMD is a bit in limbo. They have not had a proper release in over 9 years now.